src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1128,4 +1129,11 @@ private Sort mapBookingSortOrderProperties(Sort sort) {
                 .build())
             .toList();
     }
+
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
+    public VisitSummary getBookingVisitsSummary(final Long bookingId) {
+        final var visit = bookingRepository.getBookingVisitNext(bookingId, LocalDateTime.now());
```

Comment:
I know is a requirements thing; but I'm hoping this looks at the end time, e.g if a visit started 1 minute ago it would show that one until the scheduled end time.